torch/tensor.py

```diff
@@ -722,4 +722,13 @@ def _update_names(self, names, inplace):
         else:
             return super(Tensor, self).rename(names)
 
+    @property
+    def grad(self):
+        if not self.requires_grad:
```

Comment:
won't these not show in C++/JIT?  I guess that's okay for now, but maybe file an issue?

Reply:
I choose to put it here as otherwise it wouldn't know if this Tensor retains grad. Which is needed for an accurate message.